apps/etterlatte-saksbehandling-ui/client/src/components/person/uhaandtereHendelser/RelevanteHendelser.tsx

```diff
@@ -29,7 +29,9 @@ type Props = {
 const RelevanteHendelser = (props: Props) => {
   const { hendelser, disabled, grunnlag, revurderinger, sakId } = props
 
-  if (hendelser.length === 0) return null
+  if (hendelser.length === 0) {
+    return revurderinger.length > 0 ? <OpprettNyBehandling revurderinger={revurderinger} sakId={sakId} /> : null
```

Comment:
Funkar det her med revurderinger && <Opprett...>?

Reply:
Ja men Øyvind er ikke fan 😆

Reply:
haha true, ref https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
Lærer stadig!